src/Microsoft.AspNetCore.Mvc.Razor/RazorPageBase.cs

```diff
@@ -275,6 +276,23 @@ public string EndWriteTagHelperAttribute()
             return content;
         }
 
+        public virtual void PushWriter(TextWriter writer)
```

Comment:
I don't have a strong opinion either way. But I like having it be less visible initially as we can always change it later.